backend/setup.py

```diff
@@ -20,7 +20,7 @@
     "gunicorn",
     "kombu",
     "lxml",
-    "psycopg2",
+    "psycopg2-binary",
```

Comment:
This probably isn't necessary for the PR; I mostly did this to make my life easier.

Reply:
Yeah, in order to get Ibutsu to run in FedRAMP, the psycopg2 module has to be built from source, so this needs to be removed.